frontend/src/views/web/landing.html

```diff
@@ -65,9 +65,9 @@ <h4 class="grad-span">
                             <p>
                                 <span class="text-dark-black">{{main.featuredChallenge.short_description}}<span ng-if="main.isMore">...</span></span>
                             </p>
-                            <p>
-                                <span class="w-500 text-light-black">Status: </span> <span class="w-500"> In Progress</span>
-                                <span class="right">
+                            <p style="height: 15px;">
```

Comment:
Please remove `inline css`. All the css must be written in some files under the `css` folder. Also, give the `<p>` tag a class and then write it's css.